### DISCUSSION TITLE
what is the meaning of "Resample size used to grow trees"?

### DISCUSSION THREAD

**Discussion by A-Pai** [kogalur/randomForestSRC#132] [Q&A]
library(randomForestSRC)
airq.obj <- rfsrc(Ozone ~ ., data = airquality)
print(airq.obj)
![image](https://user-images.githubusercontent.com/5112397/146324255-ea128ebf-042b-45da-8af7-ffddd3a371e7.png)

**Comment by ishwaran** [comment]
This is the sample size used to grow a tree.  Remember that the forest uses random trees and part of this randomization is to select the data at random to grow the tree, which is resampling.  By default the package uses sampling without replacement `swor` with sample size .632 times original sample size.  So here the sample size used to grow a tree is on average equal to 70 which is approximately 111 x .632

You might find the getting started vignette helpful for more information about random forests.
https://luminwin.github.io/randomForestSRC/articles/getstarted.html

**Comment by A-Pai** [comment]
您的邮件我已收到                        李刚
